cmd/eskip/load.go

```diff
@@ -126,7 +127,14 @@ func printCmd(a cmdArgs) error {
 			}
 		}
 
+		if sortPredicate {
+			for _, route := range lr.routes {
+				sort.Slice(route.Predicates, func(i, j int) bool { return i < j })
```

Comment:
It seems to me such `sort.Slice` call does not actually mutate `route.Predicates` slice.
https://stackoverflow.com/questions/37695209/golang-sort-slice-ascending-or-descending 
As far as I see here to sort a slice with `sort.Slice` we need to use `i`, `j` variables as indexes to this slice. However, we compare here only indexes between each other without comparison *elements* of a slice.